fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -432,10 +437,56 @@ case $DistortionCorrection in
                 --smoothingfwhm="2" \
                 --inputdir="$WD"
         fi
-            
+
 
         ;;
 
+    ${NONE_METHOD_OPT})
+
+            log_Msg "---> No distortion correction"
+
+            log_Msg "---> FAKE WarpField"
+            ### faked warp image with three frames
+            ${FSLDIR}/bin/fslmaths ${T1wImage} -mul 0 ${WD}/WarpField.nii.gz
+            ${FSLDIR}/bin/fslmerge -t ${WD}/WarpField.nii.gz ${WD}/WarpField.nii.gz ${WD}/WarpField.nii.gz ${WD}/WarpField.nii.gz
```

Comment:
Why is a fake warpfield even necessary?  If I'm reading correctly, the eventual `convertwarp/applywarp` are within the "No distortion correction" section of the code, so let's just construct the appropriate command without the fake warpfield (which does nothing).

Reply:
It was there in case of any depencency later on in the code. If there is none, it can be skipped.

Reply:
A quick search seems to indicate that the usage of the `Warpfield` file is restricted to the SEFM correction section.